pkg/cmd/inletsoperator_app.go

```diff
@@ -50,18 +110,31 @@ func makeInstallInletsOperator() *cobra.Command {
 			return err
 		}
 
-		yamls := []string{
-			"https://raw.githubusercontent.com/inlets/inlets-operator/master/artifacts/crd.yaml",
-			"https://raw.githubusercontent.com/inlets/inlets-operator/master/artifacts/operator-rbac.yaml",
-			"https://raw.githubusercontent.com/inlets/inlets-operator/master/artifacts/operator.yaml",
+		overrides := map[string]string{}
+
+		overrides = setPlatformDepVALOverrides(overrides, command)
```

Comment:
I have another idea, what if the overrides was created inside the `setPlatformDepVALOverrides`? `getOverridesWithPlatform`?

Reply:
Fixed!